src/jit/codegencommon.cpp

```diff
@@ -1509,8 +1511,6 @@ bool CodeGenInterface::genCodeIndirAddrNeedsReloc(size_t addr)
         return true;
     }
 
-    // Else jitting.
```

Comment:
It was being moved, and seemed like an unnecessary comment to me, so instead of moving it, I deleted it. Perhaps the comment above could be updated with the otherwise we are jitting.